packages/common/src/search/types/IHubSearchResult.ts

```diff
@@ -0,0 +1,110 @@
+import { IHubGeography } from "../..";
+
+/**
+ * Standardized light-weight search result structure, applicable to all
+ * types of search results - users, groups, content, events etc
+ */
+export interface IHubSearchResult {
+  /**
+   * Access level of the backing entity
+   */
+  access: "public" | "shared" | "org" | "private";
+  /**
+   * Identifier - could be a guid or a username
+   */
+  id: string;
+  /**
+   * Hub Family: "dataset" | "map" | "document" | "content"
+   * TODO: Can we create a type for this?
+   */
+  family: string;
+  /**
+   * Actual type
+   * For Item backed results, this will be item.type
+   * Otherwise it will be "Group", "User", "Event" etc
+   */
+  type: string;
+  /**
+   * Name or title
+   */
+  name: string;
+  /**
+   * Owner; Applies to Items and Groups
+   */
+  owner: string;
```

Comment:
@dbouwman  if this only applies to items and groups (not events or people), should it be optional?